distributed/client.py

```diff
@@ -3455,6 +3455,44 @@ def restart(self, timeout=no_default, wait_for_workers=True):
             self._restart, timeout=timeout, wait_for_workers=wait_for_workers
         )
 
+    def restart_workers(self, workers: list[str]):
+        """Restart certain workers
+
+        .. note::
+
+            Only workers being monitored by nannies can be restarted
+
+        See :meth:`distributed.Nanny.restart` for more details.
+
+        Parameters
+        ----------
+        workers : list[str]
+            Workers to restart.
+
+        Examples
+        --------
+        You can get information about active workers using the following:
+
+        >>> workers = client.scheduler_info()['workers']
+
+        From that list you may want to select some workers to restart
+
+        >>> client.restart_workers(workers=['tcp://address:port', ...])
+
+        See Also
+        --------
+        Client.restart
```

Comment:
I suggest to cross reference this new method in Client.restart as well. I would also appreciate a sentence about the differences between the two methods